packages/tupaia-web/src/features/DashboardItem/DashboardItemContent.tsx

```diff
@@ -98,9 +137,19 @@ export const DashboardItemContent = ({
       </Alert>
     );
 
+  // if there is no data for the selected dates, then we want to show a message to the user
+  const showNoDataMessage = getShowNoDataMessage(report, type);
+
   return (
     <>
-      {DisplayComponent && <DisplayComponent viewContent={viewContent} isEnlarged={isEnlarged} />}
+      {showNoDataMessage ? (
+        <SmallAlert severity="info" variant="standard">
```

Comment:
There is a `NoData` component in ui-components which can be used to replace the Alert and getNoDataString util